pages/common/dnsx.md

```diff
@@ -0,0 +1,37 @@
+# dnsx
+
+> A fast and multi-purpose DNS toolkit allow to run multiple DNS queries.
+> Note: input to `dnsx` needs to be passed through `stdin` (pipe `|`) in some cases.
+> More information: <https://github.com/projectdiscovery/dnsx>.
+
+- Query the A record of a (sub)domain:
```

Comment:
What's the meaning of `-re`?

Reply:
`-re` shows the response received from the queried DNS server. For example, if a `A` record is queried, without `-re` the output will not contain the IP Address of the queried (sub)domain.

Reply:
Could you add a mnemonic to it, please?

Reply:
@vitorhcl Done